### PR TITLE
Add Vector Search section

### PR DESCRIPTION
**Pull request by andreibondarev**
I want to contribute to this list and help the maintainers, so:

- [x] I've accepted the terms of the `CC0` License;
- [x] I've added the topic `rubyml` to the contributed repository if appropriate.

### CODE REVIEW

**Comment by andreibondarev** [comment]
@arbox Whenever you get a chance to glance this over :)

File: readme.md [hunk 1]
```diff
@@ -237,6 +238,16 @@ programming languages with appropriate bindings for Ruby.
 - [lightgbm](https://github.com/ankane/lightgbm) &mdash;
   Ruby bindings for LightGBM.
   <sup>[[dep: LightGBM](#lightgbm)]</sup>
+
```
Comment (arbox) [inline readme.md:241]: Please complete every new entry with a period.

File: readme.md [hunk 2]
```diff
@@ -237,6 +238,16 @@ programming languages with appropriate bindings for Ruby.
 - [lightgbm](https://github.com/ankane/lightgbm) &mdash;
   Ruby bindings for LightGBM.
   <sup>[[dep: LightGBM](#lightgbm)]</sup>
+  
+### Vector search
+- [pgvector](https://github.com/pgvector/pgvector) &mdash;
+  Open-source vector similarity search for Postgres
```
Comment (arbox) [inline readme.md]: It is a valuable link, but not a Ruby project. If required, add it to the section with dependencies (`[[dep: ...]()]`).

**Comment by arbox** [comment]
Thank you for your contribution :) Please see my comments attached.

**Review by arbox — CHANGES_REQUESTED** [review]

**Review by arbox — APPROVED** [review]

**Comment by arbox** [comment]
Thank you!

**Pull request merged**